src/components/Billing/Invoice/AddChargeItemSheet.tsx

```
@@ -121,24 +117,31 @@ export default function AddChargeItemSheet({
+    const allItems = (response?.results as ChargeItemRead[]) || [];
+    if (allItems.length > 0) {
+      setSelectedItems(new Set(allItems.map((item) => item.id)));
     }
-  }, [items]);
+  }, [response?.results]);
 
   const handleSelectAll = (checked: boolean) => {
+    const newSelected = new Set(selectedItems);
     if (checked) {
-      setSelectedItems(new Set(items.map((item) => item.id)));
+      items.forEach((item) => newSelected.add(item.id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)